src/utilities.h

```diff
@@ -21,14 +21,15 @@
 
 #include <vector>
 #include <string>
+#include "stdio.h"
 
+//std::vector<std::string> split(const std::string& s, const std::string& delim);
 
-std::vector<CStdString> split(const CStdString& s, const CStdString& delim, const bool keep_empty = true);
+bool Str2Bool(const std::string str);
```

Comment:
Argument should read `const std::string& str`to avoid copying the value on call.